.github/workflows/tests.yml

```diff
@@ -53,21 +46,21 @@ jobs:
         with:
           go-version: '1.17' # The Go version to download (if necessary) and use.
 
-      # - uses: actions/cache@v2
-      #   with:
-      #     path: ~/go/pkg/mod
-      #     key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-      #     restore-keys: |
-      #       ${{ runner.os }}-go-
-
       - name: Clone blobber
         uses: actions/checkout@v2
 
       - name: Run tests
         run: |
           cd $GITHUB_WORKSPACE/code/go/0chain.net/ 
           CGO_ENABLED=1 go test -tags bn256 -race -coverprofile=coverage.txt -covermode=atomic ./...
-
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        with:
+          # fail_ci_if_error: true
+          files: ./code/go/0chain.net/coverage.txt
```

Comment:
@cnlangzi Anyway can avoid using absolute path's in CI/CD?

Reply:
it is generated on previous job, and use it here. There is not default value on this github action. so just leave it as it is.